beacon_node/operation_pool/src/bron_kerbosch.rs

```diff
@@ -59,18 +59,8 @@ fn compute_neigbourhoods<T, F: Fn(&T, &T) -> bool>(
 /// Produces a degeneracy ordering of a set of vertices.
 fn degeneracy_order(num_vertices: usize, neighbourhoods: &[Vec<usize>]) -> Vec<usize> {
     let mut v: Vec<usize> = (0..num_vertices).collect();
-    let mut o = vec![];
-    // move vertices from v to o in order of minimum degree
-    while !v.is_empty() {
-        let m: Option<usize> = (0..v.len()).min_by_key(|i| neighbourhoods[*i].len());
-        if let Some(i) = m {
-            o.push(v[i]);
-            v.remove(i);
-        } else {
-            break;
-        }
-    }
-    o
+    v.sort_unstable_by_key(|i| neighbourhoods[*i].len());
```

Comment:
I'm pretty sure this is what was intended, but would appreciate a double-check from you @GeemoCandama

Reply:
Yes that should be better. I don't know how I overlooked this before. I hadn't changed this from the Satalia code.